srcpkgs/brother-brscan4/template

```diff
@@ -0,0 +1,75 @@
+# Template file for 'brother-brscan4'
+pkgname=brother-brscan4
+version=0.4.4
+revision=1
+only_for_archs="i686 x86_64"
+depends="sane"
+short_desc="SANE scanner driver for brscan4-compatible printers"
+maintainer="Martijn van Buul <martijn.van.buul@gmail.com>"
+license="custom"
+homepage="http://support.brother.com/"
+repository="nonfree"
+nopie=yes
+_mylibrary="libsane-brother4.so.1.0.7"
+conf_files="/etc/opt/brother/scanner/brscan4/Brsane4.ini /etc/opt/brother/scanner/brscan4/brsanenetdevice4.cfg /etc/opt/brother/scanner/brscan4/models4/*.ini"
+
+if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
+	_debpkgid="4.amd64"
+	distfiles="http://download.brother.com/welcome/dlf006645/brscan4-${version}-${_debpkgid}.deb"
+	checksum=cd255c56e4b10125223315b64eb1205864ab8c759b6f8068c24a076f52bfc577
+	_deblibdir="usr/lib64"
+elif [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
+	_debpkgid="4.i386"
+	distfiles="http://download.brother.com/welcome/dlf006646/brscan4-${version}-${_debpkgid}.deb"
+	checksum=6292feb076a97004c5a73ce8fecd7933d29309322869c1f26f77dbe4fe92b08e
+	_deblibdir="usr/lib"
+fi
+
+# append shrinkwrap license
+distfiles+=" http://www.brother.com/agreement/English_sane/agree.html>LICENSE.html"
+checksum+=" 3434bca1936d6a5fd6afd810cde7e1876dd4d1496722b09af180278480f464f2"
+
+do_extract() {
+	ar x ${XBPS_SRCDISTDIR}/${pkgname}-${version}/brscan4-${version}-${_debpkgid}.deb
+	cd ${wrksrc}
+	tar xzpf data.tar.gz
+}
+
+do_install() {
+	# binary package makes a fine mess of things, and installs stuff in very
+	# unwieldly locations. Some of this cannot be avoided.
+
+	# deb package installs 64-bit libraries in /usr/lib64; fix this
+	cd ${wrksrc}
```

Comment:
again?

Reply:
Idem: you are already in `wrksrc`

Reply:
ping @pienjo

Reply:
Thanks for your review! I'll get back to this Real Soon Now; either later today or tomorrow.